src/verifier.rs

```diff
@@ -258,7 +253,7 @@ pub fn check(prog: &[u8]) -> Result<(), UserError> {
             ebpf::ADD64_REG  => {},
             ebpf::SUB64_IMM  => {},
             ebpf::SUB64_REG  => {},
-            ebpf::MUL64_IMM  => {},
+            ebpf::MUL64_IMM  => { check_imm_nonzero(&insn, insn_ptr)?; },
```

Comment:
Semantic difference

Reply:
Yes, stale in rbpf

Reply:
Actually, this was removed here but not in the mono repo I think.
And I would carry it forward as `MUL32_IMM` also has no such limit.

Reply:
Great, I'll take a look if we can remove it from the monorepo

Reply:
We should remove from the monorepo in s separate PR, probably with featurization, tracking here: https://github.com/solana-labs/solana/issues/17520